src/main.rs

```diff
@@ -0,0 +1,53 @@
+mod app;
+mod cli;
+mod cloud_client;
+mod errors;
+mod logger;
+mod tui;
+mod utilities;
+
+use std::{error::Error, io};
+
+static APPLICATION_NAME: &str = "csu";
+
+use crate::app::App;
+use crate::cloud_client::dropbox::client::DropboxClient;
+use crate::logger::setup_logger;
+use crate::tui::run_app;
+use crossterm::{
+    event::{DisableMouseCapture, EnableMouseCapture},
+    execute,
+    terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
+};
+use ratatui::prelude::*;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let _ = dotenvy::dotenv();
+
+    setup_logger();
+
+    let cloud_client = DropboxClient::build()?;
```

Comment:
I would make CLI the first user-facing entry point to handle all input more gracefully.